installer/test/mix_helper.exs

```diff
@@ -0,0 +1,50 @@
+Mix.shell(Mix.Shell.Process)
+
+defmodule MixHelper do
+  import ExUnit.Assertions
+
+  def tmp_path do
+    Path.expand("../../tmp", __DIR__)
+  end
+
+  defp random_string(len) do
+    len |> :crypto.strong_rand_bytes() |> Base.encode64() |> binary_part(0, len)
```

Comment:
Simple style pickiness I suppose but for piped logic I prefer one pipe per line. Mind changing it?
```suggestion
    len
    |> :crypto.strong_rand_bytes()
    |> Base.encode64()
    |> binary_part(0, len)
```